src/python/pants/backend/java/test/junit.py

```diff
@@ -118,6 +118,12 @@ async def run_junit_test(
     )
 
 
+# Required by standard test rules. Do nothing for now.
+@rule(level=LogLevel.DEBUG)
+async def setup_junit_debug_request(_field_set: JavaTestFieldSet) -> TestDebugRequest:
```

Comment:
FWIW, implementing this can be pretty straightforward: https://github.com/pantsbuild/pants/blob/3576b75b378f71196aaba9fd07925b5b2d294a7f/src/python/pants/backend/shell/shunit2_test_runner.py#L243-L255

Reply:
Understood.